src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -380,4 +377,27 @@ private static int qid(int streamIdx, int spatialIdx, int temporalIdx,
         return streamIdx * spatialLen * temporalLen
             + spatialIdx * temporalLen + temporalIdx;
     }
+
+
+    static class Pair<K, V>
```

Comment:
If you already have commons-lang3 on the classpath, could use Pair provided by it.

Reply:
i was originally using the one in javafx, which i thought was included with java 8 but i guess not...i searched around for a bit and was surprised not to find something more 'built-in'.  this code probably isn't quite ready to be merged yet, i needed to put it up so a libjitsi PR could have the tests run correctly.  hopefully we can replace this hack with something better.  i assume we don't already include commons-lang3 (didn't see it come up in auto-complete).